addons/custom-block-shape/userscript.js

```diff
@@ -0,0 +1,246 @@
+export default async function ({ addon, global, console }) {
+  var multiplier = addon.settings.get("paddingSize") / 100;
+  var BlocklyInstance = await addon.tab.traps.getBlockly();
+
+  (function (Blockly) {
+    const BlockSvg = BlocklyInstance.BlockSvg;
+    var vm = addon.tab.traps.vm;
+
+    const { GRID_UNIT } = BlockSvg;
+    var cornerSize = 1;
+    var notchSize = 1;
+
+    function updateAllBlocks() {
+      if (vm.editingTarget) {
+        vm.emitWorkspaceUpdate();
+        Blockly.getMainWorkspace().getToolbox().flyout_.workspace_.clear();
+        Blockly.getMainWorkspace().getToolbox().refreshSelection();
+      }
+    }
+
+    function applyChanges() {
+      multiplier = addon.settings.get("paddingSize") / 100;
+      cornerSize = addon.settings.get("cornerSize") / 100;
+      notchSize = addon.settings.get("notchSize") / 100;
+      BlockSvg.SEP_SPACE_Y = 2 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_X = 16 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_X_OUTPUT = 12 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_X_SHADOW_OUTPUT = 10 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_Y = 12 * GRID_UNIT * multiplier;
+      BlockSvg.EXTRA_STATEMENT_ROW_Y = 8 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_X_WITH_STATEMENT = 40 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_Y_SINGLE_FIELD_OUTPUT = 8 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_BLOCK_Y_REPORTER = 10 * GRID_UNIT * multiplier;
+      BlockSvg.MIN_STATEMENT_INPUT_HEIGHT = 6 * GRID_UNIT * multiplier;
+      BlockSvg.NOTCH_WIDTH = 8 * GRID_UNIT * multiplier;
+      BlockSvg.NOTCH_HEIGHT = 2 * GRID_UNIT * multiplier * notchSize;
+      BlockSvg.NOTCH_START_PADDING = 3 * GRID_UNIT; //* multiplier
+      BlockSvg.ICON_SEPARATOR_HEIGHT = 10 * GRID_UNIT * multiplier;
+      BlockSvg.NOTCH_PATH_LEFT =
```

Comment:
Maybe use a template string?

Reply:
?

Reply:
They mean something like

```suggestion
      BlockSvg.NOTCH_PATH_LEFT = `c 2,0 3, ${1 * notchSize} 4, ${2 * notchSize} etc`
```

Reply:
Ah, I see. It was already in the + format, so I figured it would be easier to just leave it like that.